scenarios/MLVideoInference/readme.md

```diff
@@ -0,0 +1,49 @@
+# ML video inference scenario in edge-cloud systems
+This collates the configurations and code of units and providers for ML video inference offloading scenario in an edge-cloud infrastructure.
+
+#### Requirements
+All the components of this scenario are containarized. To run this simulation, we only need:
+
+* Kubernetes or k3s (They were tested on k3s platform)
+
+
+
+#### Structure
+```
+files
+│
+└───CloudInferenceUnit/
+└───EdgeBrokerProvider/
+└───EdgeInferenceUnit/
+└───EdgePreprocessorUnit/
+└───EdgeWebServerUnit/
+└───autoscale.yml
+```
+__Every unit and provider has a separate Kubernetes manifest in their respective folder. This is the recommended approach to prevent large unmanageable k8s manifests.__
+
+#### Introducing elasticity and offloading
+The `autoscale.yml` is the manifest responsible for horizontal elasticity in the scenario. You need to fill in the respective service-name deploy it.
+
+The edge-cloud offloading logic is in the `estimate_to_cloud` method in the [EdgePreprocessorUnit/src/preprocessor.py](EdgePreprocessorUnit/src/preprocessor.py) file.You can edit it to ensure your custom logic of edge-cloud offloading
+
+#### System Overview
+The following figure presents a generic view of the ML video inference offloading scenario deployment in edge and cloud infrastructures.
```

Comment:
the deployment and services rely quite a lot of external libraries/containers. pls list them here because not only it helps people to check extra sources (not in our liability) but also improve the document of our deployment